scripts/download-unicode-files.sh

```diff
@@ -30,7 +30,7 @@ for filename in $data_files ; do
 done
 
 for filename in $emoji_files ; do
-  curl -o "$UNIDIR/$filename" "$DOWNLOAD_URL_BASE/emoji/3.0/$filename"
+  curl -o "$UNIDIR/$filename" "$DOWNLOAD_URL_BASE/emoji/5.0/$filename"
```

Comment:
We can use `/emoji/latest/`.  I didn't think that would work, since it's not in the directory listing for `emoji/`, but they use a 302 redirect.